src/java.security.jgss/share/native/libj2gss/gssapi.h

```diff
@@ -43,7 +43,9 @@
 extern "C" {
 #endif /* __cplusplus */
 
-#if defined(TARGET_OS_MAC)
+// Condition was copied from
+// Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/gssapi/gssapi.h
+#if TARGET_OS_MAC && (defined(__ppc__) || defined(__ppc64__) || defined(__i386__) || defined(__x86_64__))
```

Comment:
So for macOS/AArch64, this `if` is no longer true?

Reply:
That is according to Xcode sdk.